app/views/message.js

```diff
@@ -664,6 +668,8 @@
             options.reverse = true;
             options.remove = false;
             F.ListView.prototype.initialize.call(this, options);
+            this.ItemView.prototype.disableMessageInfo = options.disableMessageInfo;
```

Comment:
This looks wrong.  You're modifying the prototype of the ItemView class?  And doing so during an instance initialize?  If it's just an instance property it should just be `this.disableMessageInfo = options.disableMessageInfo`.

Reply:
Spent a couple hours trying to figure out how to do that. Any suggestions?